data/mods/megamax/scripts.ts

```diff
@@ -2,8 +2,8 @@ export const Scripts: ModdedBattleScriptsData = {
 	gen: 8,
 	init() {
 		const addNewMoves = (pokemonid: string, moveids: string[], tutor = false) => {
-			for (const moveid of moveids.map(toID)) {
-				this.modData('Learnsets', toID(pokemonid)).learnset[moveid] = [`8${tutor ? 'T' : 'M'}`];
+			for (const moveid of moveids.map(this.getId)) {
```

Comment:
This is gonna conflict with our other refactor ;_;

Reply:
you can wait to merge this until that one and ill just rebase